src/content/structured/accessibility/needs/motor.mdx

```diff
@@ -41,21 +46,36 @@ The following table lists examples of motor impairments in three columns (perman
 | Dyspraxia (co-ordination of movement)                      |             | General multi-tasking |
 | Muscle tremor (including Parkinson's)                      |             |                       |
 
-## Making things accessible
+## How to design for motor needs
 
-In addressing some of the design issues below, we make it easier for everyone to use our apps.
+In addressing some of the design tips below, we make it easier for everyone to use our apps.
 
 ### Clear clickable actions
 
-Make sure any buttons are clear and easily clickable. Buttons should be easy enough to click and shouldn't be too close together.
+Make sure any interactive elements are clear and easily clickable. They should be easy enough to click and shouldn't be too close together.
 
 Avoid placing lots of actions next to each other as this could increase the risk of a user taking the wrong action.
 
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={motorAccessibilityFig8}
+    imageAlt="A graphic of a yes or no option and a submit button. The clickable areas include the text of the ‘yes’ radio option and the whole button."
+    state="good"
+    caption="The cursors show a larger clickable area that includes the text for the buttons."
```

Comment:
Not sure on wording, change to something like "The cursors are positioned on a larger clickable area that includes the text for the button."